lib/src/cli/flutter_cli.dart

```diff
@@ -205,13 +205,13 @@ class Flutter {
               '--test-randomize-ordering-seed',
               randomSeed
             ],
-            if (optimizePerformance) p.join('test', '.test_runner.dart')
+            if (optimizePerformance) p.join('test', _testRunnerFileName)
```

Comment:
NIT: move `_testRunnerFileName` to the top of the file (before the classes declarations)

Reply:
Done